snippets/snip-post-mig-hook

```diff
@@ -0,0 +1,10 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+Post-migration hooks are not likely to work well with the OADP 1.3 Data Mover.
+
+The OADP 1.1 and OADP 1.2 Data Movers use synchronous processes to back up and restore application data. Because the processes are synchronous, users can be sure that any post-restore hooks start only after the persistent volumes (PVs) of the related pods are released by the persistent volume claim (PVC) of the Data Mover.
+
+However, the OADP 1.3 Data Mover uses an asynchronous process. As a result of this difference in sequencing, a post-restore hook might be called before the related PVs were released by the PVC of the Data Mover. If this happens, the pod remains in `Pending` status and cannot run the hook. The hook attempt might time out before the pod is released, leading to a `PartiallyFailed` restore operation.
```

Comment:
Any work-around for this?

Reply:
@amastbau && @weshayutin - please can we check if there is a workaround? i looked at the slack channel but could not see anything
Thanks

Reply:
@weshayutin Right, there's not really a workaround here. I think @shubham-pampattiwar is assigned the actual jira for fixing this? Or maybe I'm misremembering.

Reply:
@weshayutin && @sseago - grand so. Thanks very much for the confirmation! caps are doffed 

@shubham-pampattiwar - please can you give us a shout so we can update thanks